docs/development_suite/deploy/pipeline-based/configure-azure-pipelines.md

```diff
@@ -0,0 +1,144 @@
+---
+id: configure-azure-pipelines
+title: Deploy with Azure Pipelines
+sidebar_label: Deploy with Azure Pipelines
+---
+The Mia-Platform Console can be configured to deploy your project through Azure Pipelines.
+
+In order to enable your projects to be deployed using Azure Pipelines, in the Company Overview page you must add the Provider Azure DevOps and select the option relative to CI/CD Tool, and then follow the [standard Provider configuration](/console/company-configuration/providers/configure-provider.mdx).
+
+![Provider](./img/azure-pipelines-provider.png)
+
+This process will change the configuration of your Company in the CMS. In particular, the JSON configuration of the field `Pipelines` will change as follows:
+
+```json
+{
+  "type": "azure-pipelines",
+  "providerId": "{{PROVIDER_ID}}"
+}
+```
+
+Where `PROVIDER_ID` will be equal to the ID of the Azure Pipelines CI/CD Tool Provider that has been setup prior.
+
+For the Projects specific configuration, when you create a new one in a Company that has Azure Pipelines as CI/CD Tool Provider the Console will perform two important actions:
+- based on the [Project Template](/development_suite/company/project-templates.md) of choice, the Console will read its `azure-pipelines.yml` file and will request to Azure DevOps the creation of a new Pipeline object that will be based on this file;
+- in the CMS the `Pipelines` JSON configuration will be automatically created as follows:
```

Comment:
```suggestion
- The Project configuration `Pipelines` JSON will be defined as follows:
```

even though, this is an implementation detail that adds little value to the final user